sponsors.html

```diff
@@ -4,7 +4,19 @@
 ---
 <div class="container white-text">
   <h1 class="text-accent-4 center">Sponsor</h1>
-
+  <ul class="collapsible black">
+    <li>
+        <div class=" center collapsible-header black clemson-orange-text"><h5>Online update for Sponsors </h5></div>
+        <div class="collapsible-body">
+            <p> With CUCTF 2020 switching to an online infrastructure, all Silver, Gold, and Diamond sponsors will: </p>
+            <ul class="browser-default">
+                <li>Be provided private online meeting lobbies to connect with students and present information about their organization</li>
+                <li>Maintain sponsorship status into the CUCTF 2021 Event</li>
+            </ul>
+            <p> CU Cyber understands that interacting with students online does not match the value of meeting with students in person. To ensure that your donation gives you your full value, Silver, Gold, and Diamond sponsors will have the option to maintain their sponsor status at CUCTF 2021 with no extra cost! </p>
+            </div>
+        </li>
+  </ul>
```

Comment:
The headings have asymmetrical whitespace because they're meant between parts of the document instead of sandwiched in a collapsible header. If you're just trying for bigger text (as headers are not equivalent to making larger text), I would just add a `.update` declaration in custom.css like below along with the suggestion below that.

```
.update {
  font-size: 24px;
}
```

```suggestion
  <ul class="collapsible black">
    <li>
      <div class="collapsible-header black update clemson-orange-text">Online update for Sponsors</div>
      <div class="collapsible-body">
        <p>With CUCTF 2020 switching to an online infrastructure all Silver, Gold, and Diamond sponsors will:</p>
        <ul class="browser-default">
          <li>Be provided private online meeting lobbies to connect with students and present information about their organization</li>
          <li>Maintain sponsorship status into the CUCTF 2021 Event</li>
        </ul>
        <p>CU Cyber understands that interacting with students online does not match the value of meeting with students in person. To ensure that your donation gives you your full value, Silver, Gold, and Diamond sponsors will have the option to maintain their sponsor status at CUCTF 2021 with no extra cost!</p>
      </div>
    </li>
  </ul>
```